tests/integration/v2/slashing/slash_redelegation_test.go

```
@@ -246,24 +201,24 @@
 	createValMsg2, _ := stakingtypes.NewMsgCreateValidator(
 		goodValAddr.String(), goodValPubKey, testCoin, stakingtypes.Description{Details: "test"}, stakingtypes.NewCommissionRates(math.LegacyNewDecWithPrec(5, 1), math.LegacyNewDecWithPrec(5, 1), math.LegacyNewDec(0)), math.OneInt())
 	_, err = stakingMsgServer.CreateValidator(ctx, createValMsg2)
 	require.NoError(t, err)
 
 	// next block
-	ctx = ctx.WithBlockHeight(app.LastBlockHeight() + 1).WithHeaderInfo(header.Info{Height: app.LastBlockHeight() + 1})
-	ctx, err = simtestutil.NextBlock(app, ctx, time.Duration(1))
-	require.NoError(t, err)
+	ctx = integration.SetHeaderInfo(ctx, header.Info{Height: int64(f.app.LastBlockHeight()) + 1})
+	_, state := f.app.Deliver(t, ctx, nil)
+	_, err = f.app.Commit(state)
```

golangci-lint (1.62.2)

ineffectual assignment to err

(ineffassign)